crates/tendermint/src/lib.rs

```diff
@@ -0,0 +1,45 @@
+//! Tendermint blockchain network information types.
+//!
+//! Tendermint is a high-performance blockchain consensus engine that powers
+//! Byzantine fault tolerant applications written in any programming language.
+//! This crate provides types for representing information about Tendermint
+//! blockchain networks, including chain IDs, block IDs, and block heights.
+
+#![crate_name = "tendermint"]
+#![crate_type = "rlib"]
+#![deny(
+    warnings,
+    missing_docs,
+    trivial_casts,
+    trivial_numeric_casts,
+    unused_import_braces,
+    unused_qualifications
+)]
+#![forbid(unsafe_code)]
+#![doc(
+    html_logo_url = "https://pbs.twimg.com/profile_images/909728874560233472/cb9rWojF_400x400.jpg",
```

Comment:
Let me know if there's a better logo URL I can use. This winds up displaying in the sidebar of the Rustdoc, e.g.:

<img width="482" alt="screen shot 2018-10-22 at 3 44 58 pm" src="https://user-images.githubusercontent.com/37432020/47323628-732e4380-d611-11e8-8524-b0a0233f8ee1.png">